rebasebot/bot.py

```diff
@@ -452,6 +461,15 @@ def run(
         logging.info("rebase repository is %s", rebase_repo.clone_url)
         source_repo = gh_app.repository(source.ns, source.name)
         logging.info("source repository is %s", source_repo.clone_url)
+
+        has_manual_rebase_label = _manual_rebase_in_repo(rebase_repo)
```

Comment:
Should it be `dest_repo` instead of `rebase_repo`?

Reply:
If I understand the card for this task correctly, then we should be looking into `rebase_repo`.
The card: https://issues.redhat.com/browse/OCPCLOUD-1760